tasks/deploy/config/coordinator.sample.env

```diff
@@ -0,0 +1,9 @@
+NETWORK_NAME=<sepolia | base_sepolia | mainnet>
+NAME=<some name>_COORDINATOR
+PREFIX=<ERC4626 | StETH | RETH | EzETH>
+
+# needed for non-ERC4626, can be removed otherwise
+VAULT_SHARES_TOKEN_ADDRESS=0x...
+
+# needed for EzETH, can be removed otherwise
+EXTRA_CONSTRUCTOR_ARG=0xd94a3A0BfC798b98a700a785D5C610E8a2d5DBD8
```

Comment:
Let's document what this is for `ezETH`. It's the restake manager, so a comment like:

```
# An extra argument to be passed to the coordinator on construction. EzETH must 
# be provided with the address of the restake manager. This is unused for other 
# yield sources.
```